packages/types/src/project.ts

```diff
@@ -3,20 +3,20 @@
 
 import {ApiPromise} from '@polkadot/api';
 import {RegistryTypes} from '@polkadot/types/types';
-import {SubstrateBlock, SubstrateEvent, SubstrateExtrinsic} from './interfaces';
+import {SubstrateBlock, SubstrateEvent, SubstrateExtrinsic, AlgorandBlock, AvalancheBlock} from './interfaces';
 
 export enum SubqlDatasourceKind {
   Runtime = 'substrate/Runtime',
 }
 
 export enum SubqlHandlerKind {
-  Block = 'substrate/BlockHandler',
+  Block = 'BlockHandler',
   Call = 'substrate/CallHandler',
   Event = 'substrate/EventHandler',
```

Comment:
This look weird. 
And Substate Calls are equivalent to Avalanche Transaction. Can't we take advantage of this ?

Reply:
It s just because I hadnt did events and calls yet